demos/python_demos/deep_speech_demo/README.md

```diff
@@ -0,0 +1,65 @@
+Deep Speech Python* Demo
+===============================
+
+This is the demo application for Deep Speech algorithm, which make speech to text that are being performed on input speech audio.
+Following the below command download the pre-trained models:
+ - For UNIX*-like systems, run the following command:
+    ```sh
+    wget -O - https://github.com/mozilla/DeepSpeech/releases/download/v0.5.0/deepspeech-0.5.0-models.tar.gz | tar xvfz -
+    ```
+ - For Windows*-like systems:
+    1. Download the archive from the DeepSpeech project repository: https://github.com/mozilla/DeepSpeech/releases/download/v0.5.0/deepspeech-0.5.0-models.tar.gz.
+    2. Unzip it twice with a file archiver application.
+
+To generate the DeepSpeech Intermediate Representation (IR), provide TensorFlow DeepSpeech model to the Model Optimizer with parameters:
+
+```sh
+python3 ./mo_tf.py
+--input_model path_to_model/output_graph.pb                         \
+--freeze_placeholder_with_value input_lengths->[16]                 \
+--input input_node,previous_state_h/read,previous_state_c/read      \
+--input_shape [1,16,19,26],[1,2048],[1,2048]                        \
+--output raw_logits,lstm_fused_cell/GatherNd,lstm_fused_cell/GatherNd_1 \ 
+--disable_nhwc_to_nchw
+```
+
```

Comment:
... or get it from Open Model Zoo using `downloader.py` and `converter.py`

Reply:
Not even "or", I would say. If the model is downloadable/convertible with our scripts, the README should just tell the user to use them. No need to duplicate this information.